server/src/main/scala/com/kristofszilagyi/pipelineboard/controllers/Config.scala

```diff
@@ -17,7 +19,14 @@ object JobConfig {
   implicit val urlUserFormat: YamlFormat[UserRoot] = wrappedYamlString(s => UserRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val urlRestFormat: YamlFormat[RestRoot] = wrappedYamlString(s => RestRoot(RawUrl(Uri.parse(s))))(_.u.rawString)
   implicit val groupNameFormat: YamlFormat[GroupName] = wrappedYamlString(GroupName.apply)(_.s)
+
+  implicit val finiteDurationFormat: YamlFormat[FiniteDuration] = wrappedYamlNumber(n => FiniteDuration.apply(n.longValue(), MINUTES))(_.toMinutes)
+}
+
+object Minutes {
+  implicit val format: YamlFormat[Minutes] = yamlFormat1(Minutes.apply)
 }
+final case class Minutes(minutes: FiniteDuration)
```

Comment:
I think this should have an Int in it, not FiniteDuration. You should remove the FiniteDuration YamlFormat. And you should add a toDuration method to Minutes which converts minute to FiniteDuration

Reply:
A, makes sense, so FiniteDuration is not involved in the serializing process at all.
Done.